katas/args/rubiana/args.test.js

```diff
@@ -0,0 +1,55 @@
+import { schemaClass } from './args.js';
+import { parserClass } from './args.js';
+
+const schemaClassInstance = new schemaClass;
+test('splitInputStr() transforms the String input into an array', () => {
+    expect(schemaClassInstance.splitInputStr('-l -p 8080 -d /usr/logs')).toEqual(['-l', '-p', '8080', '-d', '/usr/logs']);
+});
+
+test('isFlag() returns true if is a flag or false if is a negative number.', () => {
+    expect(schemaClassInstance.isFlag('-p')).toBe(true);
+    expect(schemaClassInstance.isFlag('-2')).toBe(false);
+});
+
+test('validFlag() check if the flag exists in our schema', () => {
+    expect(schemaClassInstance.validFlag('p')).toBe(true);
+    expect(schemaClassInstance.validFlag('8')).toBe('Invalid! The Flag does not exist.');
+    expect(schemaClassInstance.validFlag('z')).toBe('Invalid! The Flag does not exist.');
```

Comment:
In order to keep consistency, lets make `validFlag` to always return the same type of result, a boolean. In this case, last two assertions should expect `false`